spec/classes/init_spec.rb

```diff
@@ -168,7 +168,13 @@
           })
         }
 
-        it { should_not contain_host('monkey.example.com') }
+        it {
```

Comment:
Because the catalog does contain it with ensure => absent. Previously it
did not contain it only because exported resources do not work in Travis.

On Fri, Mar 11, 2016 at 4:29 PM Garrett Honeycutt <notifications@github.com>
wrote:

> In spec/classes/init_spec.rb
> <https://github.com/ghoneycutt/puppet-module-hosts/pull/42#discussion_r55891250>
> :
>
> > @@ -168,7 +168,13 @@
> >            })
> >          }
> >
> > -        it { should_not contain_host('monkey.example.com') }
> > +        it {
>
> if the fqdn entry is false, then the catalog should not include the fqdn
> host entry. Not sure why this was changed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ghoneycutt/puppet-module-hosts/pull/42/files#r55891250>
> .
>